tests/common/plugins/conditional_mark/tests_mark_conditions_drop_packets.yaml

```diff
@@ -5,9 +5,18 @@
 #Hence, it is not dropped by default in Cisco-8000. For dropping link local address, it should be done through security/DATA ACL
 drop_packets/test_configurable_drop_counters.py::test_sip_link_local:
   skip:
-    reason: "Cisco 8000 platform does not drop SIP link local packets"
+    reason: "Cisco 8000 platform and some MLX platforms does not drop SIP link local packets"
+    conditions_logical_operator: or
     conditions:
-       - asic_type=="cisco-8000"
+      - asic_type=="cisco-8000"
```

Comment:
Suggest checking asic_type instead of HWSKU name.

Reply:
@bingwang-ms  - those test are supported and expected to pass on the default SKU, and should be skipped only on MSFT SKU where this functionality is enabled.